README.md

```diff
@@ -11,6 +11,7 @@ Overview of each file:
 - **bfcpu-freq** Display Arm core frequency.
 - **bfdracut** Create an initramfs.
 - **bffamily** Display the BlueField family for the particular board.
+- **bfgrubcheck** Checks if default grub password needs to be changed.
```

Comment:
We'll need to update the ezchip recipe as well in order to add this file into the build.

Reply:
Yes, already done.  This commit would need to go first.